openpype/hosts/maya/plugins/load/load_reference.py

```diff
@@ -45,14 +45,19 @@ def process_reference(self, context, name, namespace, options):
         except ValueError:
             family = "model"
 
-        group_name = "{}:_GRP".format(namespace)
+        if not group_name:
+            group_name = "{}:_GRP".format(namespace)
         # True by default to keep legacy behaviours
         attach_to_root = options.get("attach_to_root", True)
 
         with maintained_selection():
             cmds.loadPlugin("AbcImport.mll", quiet=True)
             file_url = self.prepare_root_value(self.fname,
                                                context["project"]["name"])
+
+            if not cmds.namespace(exists=namespace):
+                cmds.namespace(add=namespace)
+
```

Comment:
Sometimes when I referenced assets (particularly with the template feature) I had namespace issues. Basically, Maya was telling me that it couldn't reference the asset because the namespace didn't exist.

Reply:
Sadly I can't seem to reproduce the error. If I remember correctly, this happened when I loaded a reference for the first time, Maya was telling me that the namespace wasn't existing so it couldn't load the asset. But after that, if I loaded the same asset a second time, it worked because the namespace was already in the scene thanks to the first time (I don't know if that's clear).
But all of that was on the early stages of the PR, so I don't think it comes from it. Anyway, I think this is a small adding which could possibly prevent errors with namespaces. Even if the error is rare, I think it's important to take it into account.

Reply:
Actually. I'd actually argue that we don't want to ever reference into an existing namespace. So I'd actually remove this if it works without unless there's a clear problem it's solving.

Funnily enough the `maya.cmds.file` documentation states this for the `namespace` argument:

> The new namespace will be created by this command and can not already exist. The old namespace will be removed.

More reason for me to think we should have no need of generating it first?